cmd/snap/cmd_find.go

```diff
@@ -234,7 +234,7 @@ func (x *cmdFind) Execute(args []string) error {
 	}
 
 	snaps, resInfo, err := x.client.Find(opts)
-	if e, ok := err.(*client.Error); ok && e.Kind == client.ErrorKindNetworkTimeout {
+	if e, ok := err.(*client.Error); ok && (e.Kind == client.ErrorKindNetworkTimeout || e.Kind == client.ErrorKindDNSFailure) {
```

Comment:
Looking at this, should we retry on DNS errors everywhere where we retry on timeouts?

Reply:
Good point, probably, to be investigated - retry logic in httputil would be the candidate to look at, and it's used by `store.Find`. I think it's a material for possible followup though.